hugegraph-llm/src/hugegraph_llm/operators/graph_rag_task.py

```diff
@@ -117,18 +117,23 @@ def query_graphdb(
         self,
         max_deep: int = 2,
         max_items: int = 30,
+        max_v_prop_length: int = 4096,
```

Comment:
`max_v_prop_len` or `v_prop_len_max`

same as edge

Reply:
changed to `max_v_prop_len` and `max_e_prop_len`